src/CDN/CloudFrontVersion3.php

```diff
@@ -118,13 +118,23 @@ public function flushPaths(array $paths): string
                     'CallerReference' => $this->getCallerReference(),
                 ],
             ]);
-            $status = $result->get('Invalidation')['Status'];
 
-            if (false === array_search($status, self::AVAILABLE_STATUSES, true)) {
-                throw new \RuntimeException(sprintf('Unable to determine the flush status from the given response: "%s".', $status));
+            $invalidation = $result->get('Invalidation');
+            \assert(\is_array($invalidation));
+
+            $status = $invalidation['Status'] ?? null;
+
+            if (null === $status || !in_array($status, self::AVAILABLE_STATUSES, true)) {
```

Comment:
Leverage the compiler optimizations and make PHP-CS-Fixer happy.
```suggestion
            if (null === $status || !\in_array($status, self::AVAILABLE_STATUSES, true)) {
```